src/datastore/index.js

```diff
@@ -1,47 +1,36 @@
 import isString from 'lodash/isString';
 import KinveyError from '../errors/kinvey';
-import { isValidTag, clear as _clear } from './cache';
-import { NetworkStore } from './networkstore';
-import { CacheStore } from './cachestore';
+import { clear as _clear } from './cache';
+import NetworkStore from './networkstore';
+import { SyncStore } from './syncstore';
+import AutoStore from './autostore';
 
 export const DataStoreType = {
-  Cache: 'Cache',
+  Auto: 'Auto',
```

Comment:
are we going to replace `Cache`? or deprecate and remove later?

Reply:
@heyzooi I don't think there is a clean way to deprecate methods in JS. In the js-sdk, the move to "deprecate" the Cache store is a breaking change, largely because the `find()` API is removing its use of observables. So since we cannot call the method obsolete and remove it at a later time, we should just go ahead and remove the `Cache` store altogether.

Reply:
oh I see, make sense